autogpt/core/messaging/base.py

```diff
@@ -1,73 +1,35 @@
 from abc import ABC, abstractmethod
-from enum import Enum, StrEnum
 import dataclasses
-from typing import Any, List, Callable, Union
```

Comment:
Much of this will probably come back.  I'm just thinking through message broker usage as I go and wanted to start with the simplest system I could